rmw_fastrtps_cpp/CMakeLists.txt

```diff
@@ -25,6 +25,7 @@ find_package(ament_cmake REQUIRED)
 find_package(fastrtps_cmake_module REQUIRED)
 find_package(fastcdr REQUIRED CONFIG)
 find_package(fastrtps REQUIRED CONFIG)
+find_package(FastRTPS REQUIRED MODULE)
```

Comment:
Can you please describe why this line is added and what the above `find_package(CONFIG)` calls do in comparison.